source/scpp/extra/DVTableChecks.cpp

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+
+    Contains checking the order of virtual methods.
+
+    Note: This is not part of Stellar SCP code.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+
+#include "scp/SCPDriver.h"
+#include "xdr/Stellar-SCP.h"
+
+using namespace stellar;
+
+class SCPDriverDlang
+{
+    public:
+        virtual ~SCPDriverDlang() {};
+        virtual void signEnvelope () {};
+        virtual void getQSet () {};
+        virtual void emitEnvelope () {};
+        virtual void validateValue () {};
+        virtual void extractValidValue () {};
+        virtual void getValueString () {};
+        virtual void toStrKey () {};
+        virtual void toShortString () {};
+        virtual void computeHashNode () {};
+        virtual void computeValueHash () {};
+        virtual void combineCandidates () {};
+        virtual void setupTimer () {};
+        virtual void computeTimeout () {};
+        virtual void valueExternalized () {};
+        virtual void nominatingValue () {};
+        virtual void updatedCandidateValue () {};
+        virtual void startedBallotProtocol () {};
+        virtual void acceptedBallotPrepared () {};
+        virtual void confirmedBallotPrepared () {};
+        virtual void acceptedCommit () {};
+        virtual void ballotDidHearFromQuorum () {};
+};
+
+
+int checkOrderOfVM ()
+{
+    auto df0 = &SCPDriverDlang::signEnvelope;
```

Comment:
This is just comparing two C++ classes. It doesn't check the bindings. It's not what the task was about.